hawkbit-dmf/hawkbit-dmf-amqp/src/test/java/org/eclipse/hawkbit/integration/AmqpMessageDispatcherServiceIntegrationTest.java

```diff
@@ -603,4 +611,70 @@ private static boolean isDownloadAndUpdateRequest(final DmfMultiActionElement mu
                 || multiActionElement.getTopic().equals(EventTopic.DOWNLOAD_AND_INSTALL);
     }
 
+    @Test
+    @Description("Verify payload of batch assignment download and install message.")
+    public void assertBatchAssignmentsDownloadAndInstall() {
```

Comment:
you can use ParameterizedTest approach with [BATCH_DOWNLOAD, BATCH_DOWNLOAD_AND_INSTALL] topic parameter instead of two separate tests.
Additionally please add the expected events (see e.g. assignMultipleDsInMultiAssignMode)

Reply:
done